app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala

```diff
@@ -223,9 +231,9 @@ class MovementsRepositoryImpl @Inject() (
     page: Option[PageNumber] = None,
     count: Option[ItemCount] = None,
     receivedUntil: Option[OffsetDateTime] = None
-  ): EitherT[Future, MongoError, Vector[MessageResponse]] = {
+  ): EitherT[Future, MongoError, PaginationMessageSummary] = {
 
-    val projection = MessageResponse.projection
+    val messageSummaryQuery = Json.obj("$ifNull" -> Json.arr("$messageSummary", "[]"))
```

Comment:
Not sure if this is the best way ... but guess it works.

Reply:
yes,  it works so I went with this